openslides_backend/action/actions/organization/initial_import.py

```diff
@@ -0,0 +1,104 @@
+from typing import Any, Dict, Iterable, Optional, Tuple
+
+from datastore.shared.util import DeletedModelsBehaviour
+
+from ....models.checker import Checker, CheckException
+from ....models.models import Organization
+from ....shared.exceptions import ActionException, MissingPermission
+from ....shared.filters import FilterOperator
+from ....shared.interfaces.event import EventType
+from ....shared.interfaces.write_request import WriteRequest
+from ....shared.patterns import Collection, FullQualifiedId
+from ...action import Action
+from ...mixins.singular_action_mixin import SingularActionMixin
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+from ...util.typing import ActionData, ActionResultElement, ActionResults
+
+
+@register_action("organization.initial_import")
```

Comment:
You can for now just mark it as "internal", in #921 I will implement internal routes for the docker stack.

Reply:
Done.